lang/en/auth_saml2.php

```diff
@@ -33,6 +33,12 @@
 $string['auth_saml2blockredirectdescription'] = 'Redirect or display message to SAML2 logins based on configured group restrictions';
 $string['autocreate'] = 'Auto create users';
 $string['autocreate_help'] = 'If users are in the IdP but not in moodle create a moodle account.';
+$string['autologin'] = 'Auto-login';
+$string['autologin_help'] = 'Where guest access is allowed, automatically log users into Moodle with a real user account if they are logged in to the IdP (using passive authentication).';
```

Comment:
"Where guest access is allowed" this bit is a bit confusing. Guest access can mean different for different parts of moodle. What do you mean here?

Reply:
I've changed it to 'On pages that allow guest access without login' - is that clearer? Or we could remove the word 'guest' if that is confusing? Basically what I'm trying to say is, if a page calls require_login and doesn't allow guest access, then auto-login doesn't apply, it's only on the situation where the page allows access without login that autologin could happen.